src/libraries/System.CodeDom/src/System/CodeDom/Compiler/CodeGenerator.cs

```diff
@@ -971,14 +971,14 @@ protected void GenerateStatement(CodeStatement e)
             }
         }
 
-        protected void GenerateStatements(CodeStatementCollection stmts)
+        protected void GenerateStatements(CodeStatementCollection stms)
```

Comment:
@Anipik this is failing a test which expecting this name as argument name `System.CodeDom.Compiler.Tests.CodeGeneratorTests.GenerateStatements_NullStmts_ThrowsArgumentNullException` please fix the test argument name